torch/_torch_docs.py

```diff
@@ -8957,6 +8957,7 @@ def merge_dicts(*dicts):
     {input}
     dim (int, optional): the dimension to sort along
     descending (bool, optional): controls the sorting order (ascending or descending)
+    stable (bool, optional): controls the order of equivalent elements
```

Comment:
This could be more precise -- how does it "control" them? If it's false, what happens?

Reply:
Added more details about the relative ordering of indices and equivalence.